### PR TITLE
Feedback from Ketan

### PR DESCRIPTION
**Pull request by ketanketanketan**
This PR is only used for discussion and should not be merged. Instead, comments within this PR should lead to new issues which are resolved by the repository owner. This isn't to discourage raising PR's directly against changes but just a faster way to give lots of initial feedback.

### CODE REVIEW

**Comment by Indyandie** [comment]
😉

**Review by Indyandie — APPROVED** [review]
Awesome! Thanks dude this is super helpful.

File: README.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
-# Markdown Table
+# Markdown Table (Feedback)
+> This PR is only used for discussion and should not be merged. Instead, comments within this PR should lead to new issues which are resolved by the repository owner. This isn't to discourage raising PR's directly against changes but just a faster way to give lots of initial feedback.
```
Comment (Indyandie) [inline README.md:2]: Get this feedback jazz 🎷 out here!

**Review by Indyandie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ketanketanketan) [reply README.md:2]: Addressed with https://github.com/ketanketanketan/mesa/commit/5b798e2d674f2e5bfdd343e6a0bcb04ad1888f6c. Let me know if this still qualifies for a good change!

**Comment by ketanketanketan** [comment]
Because of my inactivity (🤦🏽 ) This feedback was mostly incorporated with https://github.com/Indyandie/mesa/commit/5f305c5b1f8d403cbff3b3232488c7c9c2588fee.

**Pull request closed without merging**